src/app/app.module.ts

```diff
@@ -3,12 +3,13 @@ import { BrowserModule } from '@angular/platform-browser';
 import { HttpModule }    from '@angular/http';
 
 import { AppComponent }  from './app.component';
+import { HomePageComponent } from './components/home-page/home-page.component';
```

Comment:
change the folder name to home, instead of home-page. also change the file name to home.component.ts